### PR TITLE
client: do not silence file upload failures

### PR DESCRIPTION
**Pull request by lzaoral**

### CODE REVIEW

**Comment by lzaoral** [comment]
TODO: Add a dependency on new `kobo` when this is released.

**Comment by lzaoral** [comment]
@kdudka There was a new `kobo` release so you can review this PR now.

**Comment by lzaoral** [comment]
Bodhi updates: https://bodhi.fedoraproject.org/updates/?search=kobo-0.34.0

File: osh/client/commands/shortcuts.py [hunk 1]
```diff
@@ -154,8 +155,12 @@ def fetch_results(hub, dest, task_id):
...
     """Upload file to hub, catch PermDenied exception"""
     try:
-        # returns (upload_id, err_code, err_msg)
-        return hub.upload_file(os.path.expanduser(srpm), target_dir)
+        upload_id, err_code, err_msg = hub.upload_file(os.path.expanduser(srpm), target_dir)
+        if err_code != HTTPStatus.OK:
+            raise RuntimeError(f'Uploading {srpm} failed: {err_code}: {err_msg.decode()}')
+
+        return upload_id
+
```
[the remark this answers is not in the document]
Comment (kdudka) [reply osh/client/commands/shortcuts.py:163]: I cannot see any problem with it either way but I am not a Python programmer.

[the remark this answers is not in the document]
Comment (lzaoral) [reply osh/client/commands/shortcuts.py]: I've removed the line in c10aab6.

**Pull request merged**